pkg/tracker/disputeStatus_test.go

```diff
@@ -14,10 +14,11 @@ import (
 	"github.com/tellor-io/TellorMiner/pkg/common"
 	"github.com/tellor-io/TellorMiner/pkg/db"
 	"github.com/tellor-io/TellorMiner/pkg/rpc"
+	"github.com/tellor-io/TellorMiner/pkg/util"
 )
 
 func TestDisputeString(t *testing.T) {
-	tracker := &DisputeTracker{}
+	tracker := &DisputeTracker{logger: util.SetupLogger("debug")}
```

Comment:
When possible and to be consistent lets use the constructor func to be consistent.